libs/community/langchain_community/chat_models/baidu_qianfan_endpoint.py

```diff
@@ -189,6 +189,11 @@ def validate_environment(cls, values: Dict) -> Dict:
                 default="",
             )
         )
+        return values
+
+    @root_validator(pre=False, skip_on_failure=True)
+    def post_init(cls, values: Dict) -> Dict:
```

Comment:
This logic should be moved into the pre=True. It's not validation logic (the client should usually be instantiated before init), post_init is mainly for doing consistency checks that things are initialized properly

Reply:
get.